packages/core/stellar-sdk/package.json

```
@@ -16,11 +16,14 @@
   "dependencies": {
     "@stellar/stellar-sdk": "^14.4.3",
     "@supabase/supabase-js": "^2.49.2",
+    "bignumber.js": "^9.1.2",
     "@types/jest": "^30.0.0",
     "bip39": "^3.1.0",
     "ed25519-hd-key": "^1.3.0"
   },
   "devDependencies": {
+    "@types/jest": "^30.0.0",
+    "@types/node": "^20.0.0",
```

Biome (2.1.2)

[error] The key `@types/node` was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)